e2e-tests/tests/reserve/conftest.py

```diff
@@ -31,8 +31,19 @@ def payment_key(config: ApiConfig, governance_skey_with_cli):
 
 
 @fixture(scope="session")
-def cardano_payment_key(config: ApiConfig, api: BlockchainApi, write_file):
-    payment_key_path = config.nodes_config.governance_authority.mainchain_key
+def governance_skey_with_cardano_cli(config: ApiConfig, api: BlockchainApi, write_file):
+    """
+    Securely copy the governance authority's init skey (a secret key used by the smart-contracts to authorize admin
+    operations) to a temporary file on the remote machine.
+    The temporary file is deleted after the test completes.
+
+    The skey is copied to a remote host only if both conditions are met:
+    - you call this fixture in test or other fixture
+    - tools.cardano_cli.runner.copy_secrets is set to true in the config file `<env>_stack.json`
+
+    WARNING: This fixture copies secret file to a remote host and should be used with caution.
+    """
+    payment_key_path = config.nodes_config.governance_authority.mainchain_key_local_path
```

Comment:
Actually this line is errornous. I would keep it like it was before.
I have no problem with honest docs and long name of this function.